composer.json

```diff
@@ -13,9 +13,11 @@
             "mindplay\\sql\\": "src/"
         }
     },
+    "minimum-stability": "dev",
+    "prefer-stable": true,
```

Comment:
These flags *only* apply at the project-level, e.g. only apply to this package during testing and development. We're releasing tomorrow, so I will probably remove them again then :-)